mockito/shipkit
# Ignoring contributors

Solution for issue #57
Added new configuration field and extended plugins to ignore contributors (or contributions done by contributors) defined in config.

Test I - release notes
1. install new shipkit version
2. call `./gradlew updateReleaseNotes `- release notes should contain all contributors
3. add `team.ignoredContributors = ['shipkit-org']` to **shipkit.gradle**
4. call `./gradlew updateReleaseNotes` - release notes should not contain _shipkit-org_ and its commits should not be included in the summary


Test II - contributors
1. install new shipkit version
2.  call `./gradlew fetchContributors` - `all-contributors.json` should contain all contributors
3. add `team.ignoredContributors = ['shipkit-org']` to **shipkit.gradle**
4.  call `./gradlew fetchContributors` - `all-contributors.json` should not cointain _shipkit-org_ user
